src/switchReducers.js

```diff
@@ -26,8 +26,10 @@ const isValidPattern = pattern => {
   return false
 }
 
-const validateArguments = args =>
-  args.forEach(arg => {
+const validatePairs = pairs => {
+  err(pairs.length > 0, `No Matchers were passed to switchReducers`)
```

Comment:
So we also need at least one matcher? `switchReducers(initialState)` is not a valid call anymore?

It makes sense, because if you want something that does that you can always use `always(initialState)`....

Reply:
Yes, it is done on purpose. It wouldn't make sense to use `switchReducers` for performing an `always`... even `() => initialState` would make more sense.